client/app/utils/__tests__/index.test.js

```diff
@@ -0,0 +1,21 @@
+const { randomString, getPusher } = require('../index');
```

Comment:
Let's use imports here instead, I recommend just pulling in the Utils object instead of adding new exports for each function.

```suggestion
import Utils from '../index';
```

Then you can call `Utils.randomString()` and `Utils.getPusher()`.